### PR TITLE
build 0.1.2

### PR DESCRIPTION
**Pull request by boldorider4**
* [Jira ticket](https://anaconda.atlassian.net/browse/PKG-3134)
* [Upstream repo](https://github.com/nthparty/bn254)

### CODE REVIEW

**Review by JeanChristopheMorinPerso — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+build:
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - wheel
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml:20]: setuptools not needed ?

**Review by Arishamays1 — APPROVED** [review]
LGTM

File: recipe/meta.yaml (hunk 1 above)
Comment (Arishamays1) [reply recipe/meta.yaml:20]: I'll add it! good eye!

Comment (Arishamays1) [inline recipe/meta.yaml:20]:
```suggestion
    - wheel
    - setuptools
```

**Review by Arishamays1 — CHANGES_REQUESTED** [review]

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Pull request merged**